tests/test_mapping.py

```diff
@@ -0,0 +1,98 @@
+# PyZX - Python library for quantum circuit rewriting 
+#        and optimization using the ZX-calculus
+# Copyright (C) 2018 - Aleks Kissinger and John van de Wetering
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+import random
+import sys
+if __name__ == '__main__':
+    sys.path.append('..')
+    sys.path.append('.')
+
+try:
+    import numpy as np
+    from pyzx.tensor import tensorfy, compare_tensors
+except ImportError:
+    np = None
+
+try:
+    import quimb as qu
+    import quimb.tensor as qtn
+except ImportError:
+    qu = None
+
+from pyzx.graph import Graph
```

Comment:
In this line you should do `from ..graph import Graph` instead of `pyzx.graph`, as that would make it look at installed packages instead of in the directory above it.